test/index.coffee

```diff
@@ -19,75 +19,81 @@ pullAsync = (docker, tag) ->
 			stream.once('error', reject)
 			stream.once('end', resolve)
 
-dockerUtils.getDocker({})
-.then (docker) ->
```

Comment:
These changes are to fix these tests - the `describe`s/`it`s have to be called synchronously to register with mocha